dts/bindings/display/galaxycore,gc9x01x.yaml

```diff
@@ -0,0 +1,169 @@
+# Copyright (c) 2023 Mr Beam Lasers GmbH.
+# Copyright (c) 2023 Amrith Venkat Kesavamoorthi <amrith@mr-beam.org>
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  GC9X01X display controller.
```

Comment:
Should any specific modules, tested, be listed here? It would make this driver much better searchable.

Reply:
Good point. I updated the yaml description to make this more clear.
Fixed with [8052345](https://github.com/zephyrproject-rtos/zephyr/commit/805234525391bf58e9868afa0ab5b0395bda12fe)